### PR TITLE
Update PVR API 6.2.0

### PR DESCRIPTION
**Pull request by phunkyfish**
4.1.4
- Update PVR API 6.2.0
- Transifix language update

Do not merge yet ;)

### CODE REVIEW

**Comment by phunkyfish** [comment]
API changes merged in kodi-pvr, will merge this shortly.

ios-armv7 error unrelated.

File: src/DvbData.cpp [hunk 1]
```diff
@@ -277,6 +277,9 @@ bool Dvb::GetEPGForChannel(ADDON_HANDLE handle, int iChannelUid,
     broadcast.iGenreType          = entry.genre & 0xF0;
     broadcast.iGenreSubType       = entry.genre & 0x0F;
     broadcast.iFlags              = EPG_TAG_FLAG_UNDEFINED;
+    broadcast.iSeriesNumber = EPG_TAG_INVALID_SERIES_EPISODE;
```
Comment (manuelm) [inline src/DvbData.cpp:280]: Can you please align the right side of the assignments?

File: src/DvbData.cpp [hunk 2]
```diff
@@ -580,6 +583,8 @@ bool Dvb::GetRecordings(ADDON_HANDLE handle)
     recinfo.iLastPlayedPosition = recording.lastPlayPosition;
     recinfo.iChannelUid         = PVR_CHANNEL_INVALID_UID;
     recinfo.channelType         = PVR_RECORDING_CHANNEL_TYPE_UNKNOWN;
+    recinfo.iSeriesNumber = PVR_RECORDING_INVALID_SERIES_EPISODE;
```
Comment (manuelm) [inline src/DvbData.cpp:586]: Can you please align the right side of the assignments?

**Comment by manuelm** [comment]
~~@phunkyfish thanks for ignoring my requests...~~

edit: looks like github (somehow) didn't submit me review request. So that's entirely my fault. I'm sorry.

**Comment by phunkyfish** [comment]
I will submit an update PR shortly to fix it.

**Comment by phunkyfish** [comment]
I see you made a commit already.

So I don’t need to submit a cleanup PR?

**Comment by manuelm** [comment]
No need. I've already pushed a commit a few minutes ago. Even before commenting here.

I feel sorry for falsely blaming you in the commit message but removing the commit would require a force push.

**Comment by phunkyfish** [comment]
No worries, it’s all good 😉

**Pull request merged**